dotnet/src/Connectors/Connectors.Amazon/Bedrock/Models/IBedrockModelIOService.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Nodes;
+using Amazon.BedrockRuntime.Model;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Connectors.Amazon.Models;
+
+/// <summary>
+/// Bedrock input-output service to build the request and response bodies as required by the given model.
+/// </summary>
+public interface IBedrockModelIOService
+{
+    /// <summary>
+    /// Builds InvokeModelRequest Body parameter to be serialized. Object itself dependent on model request parameter requirements.
+    /// </summary>
+    /// <param name="modelId">The model ID to be used as a request parameter.</param>
+    /// <param name="prompt">The input prompt for text generation.</param>
+    /// <param name="executionSettings">Optional prompt execution settings.</param>
+    /// <returns></returns>
+    object GetInvokeModelRequestBody(string modelId, string prompt, PromptExecutionSettings? executionSettings = null);
```

Comment:
`modelId` is needed for `MistralIOService` because the default values for the request body are different between specific mistral models, but the parameters themselves are the exact same. (I think if default values ever change between specific versions of the same model for other model providers down the line, `modelId` could be helpful again). The other option is to build a completely separate IO service class like was done for Cohere Command and Cohere Command R, but that was done because the bodies themselves are entirely different.